pykechain/models/sidebar/sidebar_manager.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, scope: 'Scope', **kwargs):
         check_type(scope, Scope, 'scope')
 
         self.scope = scope  # type: Scope
-        self._override = scope.options.get('overrideSidebar', False)  # type: bool
+        self._override = scope.options.get('overrideSideBar', False)  # type: bool
```

Comment:
could use an enum or some schema or something indeed.

Reply:
Maybe, although this is not really in line with the rest of the code. The cause for missing this typo is most likely a failure in testing, which I now included.